CodeStyle.csproj

```diff
@@ -10,6 +10,7 @@
         <PackageLicenseExpression>MIT OR Apache-2.0</PackageLicenseExpression>
         <Copyright>© Messerli Informatik AG. All rights reserved.</Copyright>
         <Description>Various analyzers bundled with opinionated configuration</Description>
+        <NoWarn>$(NoWarn);NU5105</NoWarn>
```

Comment:
Explain the warning in a comment

Reply:
7fe11ea8d0b8c2600d48f23a55f345096d53e327